{{ cookiecutter.package_name }}/pyproject.toml

```
@@ -27,11 +27,35 @@ dependencies = [
 ]
 dynamic = ["version"]
 
+# Okay strip the whitespace motherfucker!
+[project.urls]
+{% if cookiecutter.project_url -%}
+Homepage = "{{ cookiecutter.project_url }}"
+{%- else -%}
+Homepage = "https://sunpy.org"
+{%- endif -%}
+
+{%- if cookiecutter.sourcecode_url %}
```

Would this not have the same effect as the blank line without the blank line?
```suggestion
{% if cookiecutter.project_url -%}
Homepage = "{{ cookiecutter.project_url }}"
{% else -%}
Homepage = "https://sunpy.org"
{% endif -%}
{%- if cookiecutter.sourcecode_url %}
```